README.md

```diff
@@ -11,18 +10,20 @@ build pipeline that is more comprehensible, reproducible, and extendable.
 See the `osbuild(1)` man-page for details on how to run osbuild, the definition
 of the pipeline description, and more.
 
-### Project
+## Project
 
  * **Website**: <https://www.osbuild.org>
  * **Bug Tracker**: <https://github.com/osbuild/osbuild/issues>
  * **IRC**: #osbuild on [Libera.Chat](https://libera.chat/)
```

Comment:
Shall we deprecate IRC? The room can stay, but I would maybe remove it from our docs.

Reply:
Agree!